scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -984,6 +991,7 @@ protected final void handshakeStarted() throws HandshakeException {
 	}
 
 	protected final void sessionEstablished() throws HandshakeException {
+		amendPeerPrincipal();
```

Comment:
I don't kown if `amendPeerPrincipal` shoud be in or out the `if block` :thinking: